From: Hilko Bengen <bengen@debian.org>
Date: Tue, 8 Dec 2020 22:42:28 +0100
Subject: Add hardening flag

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8367757..8937219 100644
--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ ifneq (,$(findstring yes,$(CAPSTONE_X86_ATT_DISABLE)))
 CFLAGS += -DCAPSTONE_X86_ATT_DISABLE
 endif
 
-CFLAGS += -fPIC -Wall -Wwrite-strings -Wmissing-prototypes -Iinclude
+CFLAGS += -fPIC -Wall -Wwrite-strings -Wmissing-prototypes -Iinclude -D_FORTIFY_SOURCE=2
 
 ifeq ($(CAPSTONE_USE_SYS_DYN_MEM),yes)
 CFLAGS += -DCAPSTONE_USE_SYS_DYN_MEM
